modules/migration-viewing-migration-plan-resources.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/troubleshooting-3-4.adoc
+// * migration-toolkit-for-containers/troubleshooting-mtc.adoc
+
+[id="migration-viewing-migration-plan-resources_{context}"]
+= Viewing migration plan resources
+
+You can view the resources of a migration plan to monitor the progress of a live migration or to troubleshoot a failed migration. You use the {mtc-short} web console to copy a command to your clipboard. Then, you run the command from the command line interface (CLI).
+
+If you are troubleshooting a failed migration, start with a high-level resource that has failed and then work down the resource tree, towards the lower-level resources.
+
+.Procedure
+
+. In the {mtc-short} web console, click *Migration Plans*.
+. Click the *Migrations* number next to a migration plan to view the *Migrations* page.
++
+The *Migrations* page displays the migration types associated with the migration plan, for example, *Stage* or *Cutover* for warm migration.
```

Comment:
@apinnick I am confused about  *Cutover*

Reply:
"Cutover" is the second "type" of warm migration. The Type field will show "stage" or "cutover" for warm migration, if they have been run.

Reply:
@apinnick for MTC, it just has 3 types of migration, "stage", "migration", "rollback", I never heard the term "warm migration" on MTC.

Reply:
You're correct. Thanks for catching that.